py_mmd_tools/mmd_to_nc.py

```diff
@@ -0,0 +1,135 @@
+"""
+Tool for parsing an XML metadata file, following the MET Norway Metadata specification (MMD),
+ and updating an NetCDF file, following the Attribute Convention for Data Discovery (ACDD).
+
+License:
+
+This file is part of the py-mmd-tools repository
+<https://github.com/metno/py-mmd-tools>.
+
+py-mmd-tools is licensed under the Apache License 2.0
+<https://github.com/metno/py-mmd-tools/blob/master/LICENSE>
+"""
+
+import netCDF4 as nc
+import lxml.etree as ET
+
+
+class Mmd_to_nc(object):
+
+    def __init__(self, mmd_product, nc_file):
+        """Class for updating a NetCDF file that is compliant with the CF-conventions and ACDD
+        from an MMD XML file.
+        Args:
+            mmd_product (str): Input MMD xml file.
+            nc_file (pathlib): Nc file to update.
+        """
+        super(Mmd_to_nc, self).__init__()
+        self.nc = nc_file
+        self.mmd = mmd_product
+        return
+
+    def update_nc(self):
+        """
+        Update a netcdf file global attributes.
+        """
+
+        tree = ET.parse(self.mmd)
+
+        # Open netcdf file for reading and appending
+        with nc.Dataset(self.nc, 'a') as f:
+
+            acdd = {}
+            ns = {'xml': 'http://www.w3.org/XML/1998/namespace',
+                  'mmd': 'http://www.met.no/schema/mmd'}
+
+            # ACDD - Highly recommended attributes
+
+            acdd['title'] = tree.find('mmd:title[@xml:lang="en"]', namespaces=ns).text
```

Comment:
There should be a way to use `mmd_elements.yaml`. By looping the content of an MMD XSD file, it should be possible to simplify and populate the `acdd` dict automatically, e.g., something like:
```
for element in xsd:
    acdd[mmd_yaml[element_name.acdd]] = tree.find(element).text

f.setncatts(acdd)
```
This way, we make sure that the tool will not fail when we update MMD or the translations...

Reply:
I tried to do so by looping through the mmd elements on the translation document. But some parts are still hard coded as there is not a 1-1 match between mmd and ACDD.